app/helpers/application_helper.rb

```diff
@@ -1,2 +1,23 @@
 module ApplicationHelper
+  def posts_per_page
+    3
+  end
+
+  def author_posts_more_than?(user, limit_ = 2)
+    user.posts.count > limit_
+  end
+
+  def flash_messages_for(object)
+    render(partial: 'layout/flash', locals: { object: })
+  end
```

Comment:
- [ ] Nice job on the blog app, however, the app is crashing because of line 11, you are passing the flash message as an object inside a parameter but it will only work if you make a string.

- **before**

![Capture web_18-6-2022_204844_localhost](https://user-images.githubusercontent.com/61562626/174455238-fc310bb4-7780-4e54-91a6-e16b3d1c3c08.jpeg)

![Capture d’écran 2022-06-18 205318](https://user-images.githubusercontent.com/61562626/174455287-2c62bcd2-3db6-4b4f-97cd-135e94c34b44.png)

- **After**

![Capture d’écran 2022-06-18 205343](https://user-images.githubusercontent.com/61562626/174455295-fd18cbd1-741f-4393-911d-a720ad25ac2e.png)
![Capture web_18-6-2022_205252_localhost](https://user-images.githubusercontent.com/61562626/174455296-ad242d2d-e908-4ff3-a6ae-501379bfdeb3.jpeg)